Move Imageviewer from scifio to the Show command

gab1one opened the pull request:
The Imgviewer will be removed from scifio in the near future, and has
found a new home here.

ctrueden commented:
Can you please make the style cleanup its own commit?

Also, are you sure you applied the correct style cleanup rules?

Commit:
Move Imageviewer from scifio to the Show command

The Imgviewer will be removed from scifio in the near future, and has
found a new home here.

gab1one commented:
Sorry about the styling stuff, that should not have been committed, I cleaned the commit up.

ctrueden commented:
Cool, thanks. I think later, we can probably remove this inner class in favor of just launching an ImageJ. It would be a nicer and more consistent viewer, and greatly reduce the amount of code here too.
